src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector128.cs

```diff
@@ -38,9 +38,9 @@ public static unsafe Vector128<byte> Create(byte value)
             return Unsafe.AsRef<Vector128<byte>>(pResult);
         }
 
-        /// <summary>Creates a new <see cref="Vector128{double}" /> instance with all elements initialized to the specified value.</summary>
+        /// <summary>Creates a new <see cref="Vector128{Double}" /> instance with all elements initialized to the specified value.</summary>
```

Comment:
:bulb: There are a bunch of additional cases in this file where the type parameter name doesn't match the symbol definition.